index.php

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="web.css">
+</head>
+<body>
+        <nav>
+            <div class="log">
+                <img src="images/log.png" class="tif">
+
+            </div>
+            <p class="logo" >ACADEMICS</p>
+
+            <ul>
+                <li><a href="web.html">HOME</a></li>
+                <li><a href="course.html" class="white-menu">COURSES</a></li>
+                <li><a href="health.html">HEALTH</a></li>
+                <li><a href="event.html">EVENT</a></li>
+                <li><a href="gallery.html">GALLERY</a></li>
+                <li><a href="contact.html">CONTACT</a></li>
+            </ul>
+            <button class="RF">
+                <a href="sign_up.html" class="fr">APPLY NOW</a>
+            </button>
+        </nav>
```

Comment:
Great job so far

- [ ] Please create a single file (header.php) to handle the navbar and use the PHP include() method to add the header into your different pages.